examples/01a-Getting-started.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>Can we have all required imports in the beginning of the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>